app/js/lib/foxbox/recipes.js

```diff
@@ -54,9 +54,8 @@ export default class Recipes {
       'POST',
       { getters: [{ kind: 'ThinkerbellRuleSource' }] }
     )
-    .then((services) => {
       // Mark getters and setters with more friendly names.
-      return services.map((service) => {
+      .then((services) => services.map((service) => {
```

Comment:
Yep, sounds good! (I think eventually we'll be able to make it ```error``` again, since big ```map``` callbacks almost asking for a dedicated function anyway :))